posthog/src/main/java/com/posthog/PostHog.kt

```diff
@@ -757,11 +757,20 @@ public class PostHog private constructor(
         }
 
         if (config?.sendFeatureFlagEvent == true && shouldSendFeatureFlagEvent) {
+            var flagDetails = remoteConfig?.getFlagDetails(key)
+
+            val requestId = remoteConfig?.getRequestId()
```

Comment:
just a nit but you could do a single check for `remoteConfig` or use `?.let`.
right now this will be 2 `if` conditions if `remoteConfig` is null or not